File: app/src/main/java/com/example/myappcontacts/data/providers/database/databaseutils/ContactsDbSchema.java
package com.example.myappcontacts.data.providers.database.databaseutils;

public class ContactsDbSchema {
    public static final class ContactsTable {
        public static final String NAME = "contacts";

        public static final class Cols {
            public static final String UUID = "uuid";
            public static final String FIRST_NAME = "firstName";
            public static final String LAST_NAME = "lastName";
            public static final String COMPANY_NAME = "companyName";
            public static final String TEL_NUMBER = "telNumber";
            public static final String PHOTO_URI = "photoUri";
            public static final String ADDRESS = "address";
        }
    }
}

File: app/src/main/java/com/example/myappcontacts/data/providers/database/databaseutils/ContactsDbSchema.kt
package com.example.myappcontacts.data.providers.database.databaseutils

class ContactsDBSchema {
    object ContactsTable {
        const val NAME = "contacts"

        object Cols {
            const val UUID = "uuid"
            const val FIRST_NAME = "firstName"
            const val LAST_NAME = "lastName"
            const val COMPANY_NAME = "companyName"
            const val TEL_NUMBER = "telNumber"
            const val PHOTO_URI = "photoUri"
            const val ADDRESS = "address"
        }
    }
}